### PR TITLE
Sfp 106 nl

### PR DESCRIPTION
**Pull request by ToonVanDaele**
## Description
Add new protocol sfp-106-nl. Imported from docx

## Task list

Steps by contributor:

- [x] Add description to this pull request (under "## Description")
- [x] Submit the pull request by clicking 'create pull request'
- [x] Mark the pull request as draft
- [x] Check whether the version number in the `index.Rmd` yaml section is of format `yyyy.nn.dev`
- [x] In case of a protocol created from a pre-existing `docx` protocol, check if all sections comply with current template for a new protocol
- [x] Add further commits if needed and push them to GitHub
- [x] Update the protocol-specific `NEWS.Rmd`
- [x] Mark the pull request as 'ready for review'

Review steps for the author(s):

- [x] Add reviewers, at least one subject-matter specialist and one administrator
- [x] Wait for review comments and address them
- [x] Iterate until reviewer approvals (merging the pull request will be done by an administrator)

To be done by an administrator after review, but before merging the pull request:

- [ ] Update the `.zenodo.json` file (authorship will extend as protocols are added)
- [ ] Bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`.
- [ ] Update the repo `NEWS.md` file

To be done by an administrator after merging this pull request:

- [ ] Add general tag to the merge commit (`protocols-YYYY.NN`)  (see [release model](README.md#release-model))
- [ ] Add specific tag to the merge commit (`<protocol-code>-YYYY.NN`)  (see [release model](README.md#release-model))
- [ ] Check whether continuous integration proceeded without problems
- [ ] [Create a new release](https://github.com/inbo/protocols/releases/new) based on a general tag
- [ ] Check success of publication steps at protocols.inbo.be and at Zenodo (records both source and rendered)

### CODE REVIEW

**Comment by hansvancalster** [comment]
@ToonVanDaele Thanks for adding these protocols. I will review this later, but I noticed something was wrong with the name / orcid in the yaml so I fixed it here. See [390f668](https://github.com/inbo/protocolsource/pull/54/commits/390f668d322aacead178f285214398ca3d0f423e)
can you check this for the other PR as well?

**Comment by hansvancalster** [comment]
I noticed figures were not rendered in the artifact probably due to the use of backslashes in the relative paths to the media folder. I changed this in [f940659](https://github.com/inbo/protocolsource/pull/54/commits/f940659d65b24a59f30e8267fd718c33daaf17cd). 

@ToonVanDaele just to be sure: I think these backslashes were not introduced by you. They were there after you used the from_docx conversion function? If so, than this is a bug in protocolhelper that we need to address.

Can you also check this in your other PRs? Search for %5C (which is a backslash) and replace by /.

**Comment by ToonVanDaele** [comment]
@hansvancalster indeed, backslashes %5C were introduced by the conversion function. It worked fine for me to render the html page with the images, the forward slash works fine too. %5C is now replaced with forward slash in sfp-104-nl en sfp-105-nl branches.

Orcid is corrected in sfp-104-nl and sfp-105-nl branches too

**Comment by hansvancalster** [comment]
> @hansvancalster indeed, backslashes %5C were introduced by the conversion function. It worked fine for me to render the html page with the images, the forward slash works fine too. %5C is now replaced with forward slash in sfp-104-nl en sfp-105-nl branches.

Thanks, I'll make an issue so we can fix this in protocolhelper. We need to fix it, because the protocols will be rendered on Linux machines (for publication on the protocols website and also for the [artifact](https://github.com/inbo/protocolsource/actions/runs/2306089316) in each PR). Linux machines do not recognize Windows style paths with backslashes.

File: src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/02_onderwerp.Rmd [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```{=html}
+<!--
+Doelstelling: omschrijving in woorden waarvoor het protocol dient.
+Voorbeelden wat er met de gegevens kan gedaan worden.
+Probeer de doelstelling zo veel mogelijk generiek te beschrijven en vermijd projectspecifieke doelstellingen.
+Toepassingsgebied: bespreek hier onder welke condities het protocol kan gebruikt worden.
+-->
+```
+Deze richtlijn heeft betrekking op peilmetingen in een peilbuis, geplaatst volgens het protocol beschreven in de richtlijn SVP-104 (richtlijnen voor het plaatsen van een peilbuis)..
```
Comment (hansvancalster) [inline src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/02_onderwerp.Rmd:30]: Dus dit protocol is afhankelijk van https://github.com/inbo/protocolsource/pull/52. Dat betekent dat we eerst die PR moeten afwerken. Daarna kunnen we naar dat protocol verwijzen. Er is `protocolhelper::add_dependency()` om ons daarbij te helpen.

File: src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/04_principe.Rmd [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<!--
+Korte omschrijving van de principes waarop het protocol gebaseerd is.
+Bv principe achter het nemen van mengmonsters, principe achter visueel inschatten van de bedekking van een plant, ...
+-->
+```
+## Manuele peilmeting
+
+De peilmeting is de afstand tussen het referentiepunt (de bovenrand van de peilbuis zonder dop) en het wateroppervlak in de peilbuis (figuur \@ref(fig:Figuur1)).
+Omdat deze meting naar beneden gericht is wordt dit met een negatief cijfer genoteerd.
+Komt bij een overstroming het peil boven het meetpunt, dan moet een positieve waarde genoteerd worden.
```
Comment (hansvancalster) [inline src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/04_principe.Rmd:13]: "meetpunt" is onduidelijk. Bedoel je hiermee de top van de peilbuis?

File: src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/07_stappenplan.Rmd [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Gedetailleerde omschrijving van alle stappen die doorlopen moeten worden om het protocol uit te voeren.
+Subtitels gebruiken om elke stap te omschrijven.
+-->
+```
+### Meetlint met 'klokje'
+
+Voor ondiepe grondwaterstanden is deze methode zeer praktisch en betrouwbaar.
+De lintmeter met het klokje wordt neergelaten in de peilbuis.
+Wanneer het klokje het\
+wateroppervlak raakt hoor je een 'plop' geluid.
```
Comment (hansvancalster) [inline src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/07_stappenplan.Rmd:16]:
```suggestion
Wanneer het klokje het wateroppervlak raakt hoor je een 'plop' geluid.
```

File: src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/07_stappenplan.Rmd [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+De manuele referentie peilmeting moet daarom gebeuren vóór de druksensor in de peilbuis neergelaten wordt.
+Bij het bovenhalen van de sensor (voor uitlezen of definitief verwijderen) moet eerst de manuele referentie peilmeting uitgevoerd worden.
+Voor een langere reeks metingen met een druksensor zijn meerdere referentiemetingen per jaar aangewezen (minimaal 2).
+
+Een defect of foute instelling van de druksensor wordt pas opgemerkt bij het uitlezen.
+De druksensor wordt daarom minimaal 2 maal per jaar uitgelezen.
+
+## Registratie en bewaring van resultaten
+
+De peilmetingen worden in het veld genoteerd (zie bijlage 1 en 2) en achteraf in Watina ingevoerd.
```
Comment (hansvancalster) [inline src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/07_stappenplan.Rmd:52]: Deze verwijzingen kan je vervangen door kruisverwijzingen naar de secties. Zie https://bookdown.org/yihui/bookdown/cross-references.html

File: src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/04_principe.Rmd (hunk 2 above)
Comment (ToonVanDaele) [reply src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/04_principe.Rmd:13]: Inderdaad, dit moet 'referentiepunt' zijn. De zin is aangepast om dit op te helderen

**Review by hansvancalster — APPROVED** [review]

**Review by hansvancalster — APPROVED** [review]

**Comment by florisvdh** [comment]
> > Linux machines do not recognize Windows style paths with backslashes.
> 
> Of course. In Unix (way before Windows came up) backslash was used as escape character. So this has not changed on Unix shells, and in various languages. Moreover, just using forward slashes in path strings in R will work on any platform. See also https://stackoverflow.com/a/7963304, where it is stated '_what's not widely known is that DOS/Windows is also perfectly happy to use the forward slash (/) as its path separator. Try it._'

FYI: in the R for Data Science book, you can [read](https://r4ds.hadley.nz/workflow-scripts#relative-and-absolute-paths): '_we recommend always using the Linux/Mac style with forward slashes_'.

**Pull request merged**